cuda_core/docs/source/api.rst

```diff
@@ -1,4 +1,4 @@
-.. module:: cuda.core.experimental
+.. currentmodule:: cuda.core.experimental
```

Comment:
I thought this does not need to be changed?

Reply:
you are correct, the issue was actually a missing pip install. fixed